presto-main/src/test/java/com/facebook/presto/spiller/TestGenericPartitioningSpiller.java

```diff
@@ -74,6 +74,7 @@ public void setUp()
         FeaturesConfig featuresConfig = new FeaturesConfig();
         featuresConfig.setSpillerSpillPaths(tempDirectory.toString());
         featuresConfig.setSpillerThreads(8);
+        featuresConfig.setSpillMaxUsedSpaceThreshold(1.0);
```

Comment:
Not sure we want this. Having tests fill remaining free space on disk can be ... not nice.

(if we want this, then `TestDistributedSpilledQueries` calls for the same treatment)

Reply:
We do this for the other tests that create the spiller directly. Reserving 10% means I need 100GB free to run tests on my laptop.

We can also change for distributed tests but those are disabled by default so I’m not as concerned.